public/src/supported-events-policy.md

```diff
@@ -0,0 +1,48 @@
+<style type="text/css">
+  a[href]:after { content : "" }
+</style>
+
+Version: 1.0
+Updated: July 18, 2022
+
+# Speedcubing Canada Supported Events Policy
+
+Uses of the words “must”, “must not”, “should”, “should not” and “may” match [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+All dollar amounts discussed are in Canadian dollars.
+
+Speedcubing Canada’s Board of Directors reserves the right tomodify this document with a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
+
+A Speedcubing Canada Supported Event is an event, usually an official World Cube Association (WCA) competition in Canada, that is organized by or in partnership with Speedcubing Canada under the following conditions:
+
+## Recognition Process
+
+- Written notice from Speedcubing Canada’s Board of Directors **must** be obtained to officially recognize a Speedcubing Canada Supported Event.
+- Individuals seeking to run a Speedcubing Canada Supported Event **must** email [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) to request that their event be recognized as a Speedcubing Canada Supported Event.
+  - The email **must** include a detailed, itemized budget for the event, including all expenses and sources of income.
+    - The budget _should_ be balanced and _should_ include a not insignificant margin to account for unexpected costs and circumstances.
```

Comment:
Do you have a particular amount or percentage for the "not insignificant margin"?